Web3 Patching Typo

**Describe the bug**

The npm script doesn't return zero upon exit.

**To Reproduce**

Re-run ci.sh twice

**Expected behavior**

ci.sh works

**Logs/Stack traces**

```sh
Starting the SDK server ... REDACTED_PATH/blockchain-integration-framework/examples/simple-asset-transfer/fabric/scripts
7472
Initializing the network ...
[2019-11-18T18:36:08.522] [INFO] init-network - Naima enrolled Successfully
[2019-11-18T18:36:09.160] [INFO] init-network - Mike enrolled Successfully
[2019-11-18T18:36:10.229] [INFO] init-network - Hugo enrolled Successfully
[2019-11-18T18:36:10.971] [INFO] init-network - Luca enrolled Successfully
[2019-11-18T18:36:11.257] [INFO] init-network - Channel 'mychannel' created Successfully
[2019-11-18T18:36:21.430] [INFO] init-network - Successfully joined peers in organization Org1 to the channel:mychannel
[2019-11-18T18:36:21.786] [INFO] init-network - Successfully installed chaincode
[2019-11-18T18:36:31.852] [INFO] init-network - Successfully joined peers in organization Org2 to the channel:mychannel
[2019-11-18T18:36:32.160] [INFO] init-network - Successfully installed chaincode
[2019-11-18T18:38:27.578] [INFO] init-network - Successfully instantiate chaincode in organization Org1 to the channel 'mychannel'
+ cd ./quorum/api/
+ rm -rf ./node_modules/websocket/.git/
+ npm install

> @hyperledger-labs/blockchain-integration-framework.examples.simple-asset-transfer.quorum-api@0.1.0 postinstall REDACTED_PATH/blockchain-integration-framework/examples/simple-asset-transfer/quorum/api
> patch --verbose --force ./node_modules/web3-core-helpers/src/formatters.js ./web3_timestamp_fix.patch

Hmm...  Looks like a normal diff to me...
patching file ./node_modules/web3-core-helpers/src/formatters.js
Using Plan A...
Hunk #1 FAILED at 266.
1 out of 1 hunk FAILED -- saving rejects to file ./node_modules/web3-core-helpers/src/formatters.js.rej
done
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @hyperledger-labs/blockchain-integration-framework.examples.simple-asset-transfer.quorum-api@0.1.0 postinstall: `patch --verbose --force ./node_modules/web3-core-helpers/src/formatters.js ./web3_timestamp_fix.patch`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @hyperledger-labs/blockchain-integration-framework.examples.simple-asset-transfer.quorum-api@0.1.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/multipass/.npm/_logs/2019-11-19T02_38_40_111Z-debug.log
```

**Screenshots**

If applicable, add screenshots to help explain your problem.

**Cloud provider or hardware configuration:**
Are you running the software on a dev machine or somewhere in the cloud?

**Operating system name, version, build:**

Use this command: `printf "$(uname -srm)\n$(cat /etc/os-release)\n"`

For example

```sh
Linux 4.15.0-70-generic x86_64
NAME="Ubuntu"
VERSION="18.04.3 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.3 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic
```

**BIF release version or commit (git rev-parse --short HEAD):**

76a97af

**BIF Plugins/Connectors Used**

Fabric, Quorum

**Additional context**

This was a bug introduced while re-working code in response to reviews, silly mistake of mine.

--- a/examples/simple-asset-transfer/quorum/api/package.json
+++ b/examples/simple-asset-transfer/quorum/api/package.json
@@ -11,7 +11,7 @@
     "start": "node index.js",
     "upgrade-interactive": "npm-check --update",
     "test": "cross-env NODE_ENV=test ./node_modules/.bin/mocha --ui bdd --reporter spec --colors routes/**/*.test.js --recursive --exit",
-    "postinstall": "patch --verbose --force ./node_modules/web3-core-helpers/src/formatters.js ./web3_timestamp_fix.patch"
+    "postinstall": "patch --verbose --force ./node_modules/web3-core-helpers/src/formatters.js ./web3_timestamp_fix.patch || true"
   },
   "dependencies": {
     "body-parser": "^1.18.3",

